literature.bib

```diff
@@ -258,6 +291,115 @@ @inproceedings{KHS+:VaMoS24
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2023 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@techreport{AKS:TR23,
+	author = {Adamy, Rick and Kuiter, Elias and Saake, Gunter},
+	title = {{Exploiting Structure: A Survey and Analysis of Structures and Hardness Measures for Propositional Formulas}},
+	doi = {10.32388/7u1pfg},
+	month = SEP,
+	year = 2023,
+	institution = UniMagdeburg,
+	archiveprefix = {{Qeios}},
+	eprint = {7u1pfg}
+}
+
+@inproceedings{ALR+:VaMoS23,
+	author = {Acher, Mathieu and Lesoil, Luc and Randrianaina, Georges Aaron and T\"{e}rnava, Xhevahire and Zendra, Olivier},
+	title = {{A Call for Removing Variability}},
+	booktitle = VaMoS20,
+	publisher = ACM,
+	address = NY,
+	location = {Odense, Denmark},
+	pages = {82--84},
+	isbn = {9798400700019},
+	doi = {10.1145/3571788.3571801},
+	year = 2023,
+	numpages = 3
+}
+
+@inproceedings{BFW:SAT23,
+	author = {Biere, Armin and Froleyks, Nils and Wang, Wenxi},
+	title = {{CadiBack: Extracting Backbones with CaDiCaL}},
+	booktitle = SAT,
+	pages = {3:1--3:12},
+	series = {Leibniz International Proceedings in Informatics (LIPIcs)},
+	isbn = {978-3-95977-286-0},
+	issn = {1868-8969},
+	year = 2023,
+	volume = 271,
+	editor = {Mahajan, Meena and Slivovsky, Friedrich},
+	publisher = SchlossDagstuhl,
+	address = Dagstuhl,
+	doi = {10.4230/LIPIcs.SAT.2023.3}
+}
+
+@misc{C:23,
```

Comment:
We do have some websites at the end of the file but typically use a more stable version of the key. Would you update the key every year it is used again?